### PR TITLE
Use log4j to log well-formatted messages

### PR DESCRIPTION
**Pull request by topnessman**
View the format from this link: 
https://paste.ofcode.org/BfVybJWVX5VxUUt2qPpQWS
Original format:
https://paste.ofcode.org/Y6aqK5aFUTBPFHrsmk6UfJ#

### CODE REVIEW

File: src/checkers/inference/InferenceCli.java [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+
 import java.io.IOException;
 import java.util.logging.ConsoleHandler;
 import java.util.logging.Handler;
-import java.util.logging.Logger;
 import java.util.logging.Level;
-
-import plume.Option;
-import plume.Options;
+import java.util.logging.Logger;
```
Comment (CharlesZ-Chen) [inline src/checkers/inference/InferenceCli.java]: JUI imports seems doesn't needed anymore. Can we remove unused imports?

File: src/checkers/inference/InferenceAnnotatedTypeFactory.java [hunk 2]
```diff
@@ -1,5 +1,22 @@
 package checkers.inference;
 
+import checkers.inference.dataflow.InferenceAnalysis;
```
Comment (CharlesZ-Chen) [inline src/checkers/inference/InferenceAnnotatedTypeFactory.java]: Why change the original order of imports? It makes hard to see which imports are newly added, which are just reordered.

File: src/checkers/inference/InferenceMain.java [hunk 3]
```diff
@@ -290,9 +287,9 @@ private InferrableChecker getRealChecker() {
                         InferenceOptions.checker, true, ClassLoader.getSystemClassLoader()).newInstance();
                 realChecker.init(inferenceChecker.getProcessingEnvironment());
                 realChecker.initChecker();
-                logger.finer(String.format("Created real checker: %s", realChecker));
+                logger.info(String.format("Created real checker: %s", realChecker));
```
Comment (CharlesZ-Chen) [inline src/checkers/inference/InferenceMain.java]: Based on the log4j official site: `FINER` in JUL are equal to `TRACE` in log4j. Can we keep the original log level?

here is the convert log level table from JUL to log4j:

https://logging.apache.org/log4j/2.x/log4j-jul/

**Comment by CharlesZ-Chen** [comment]
@topnessman  Mier, ping here. What is the current status of this PR?

**Comment by topnessman** [comment]
Close it because not so important

**Pull request closed without merging**